nautobot/extras/templates/extras/inc/jobresult.html

```diff
@@ -58,8 +57,3 @@
     </div>
     </form>
     {% table_config_form log_table %}
-{% else %}
-    <div class="alert alert-danger" role="alert">
-        No table of job logs was provided, perhaps you are using an out-of-date plugin?
-    </div>
-{% endif %}
```

Comment:
Don't remove this - this is useful to flag for plugins that might be using this template (such as `nautobot-ssot`) and need to be updated to handle the 1.2.0 changes to this template.

Reply:
Cool.